alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/MonitorRunResult.kt

```diff
@@ -77,10 +77,14 @@ data class TriggerRunResult(
     val actionResults: MutableMap<String, ActionRunResult> = mutableMapOf()
 ) : ToXContent {
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
+        var msg = error?.message;
+        if(error is ScriptException){
```

Comment:
tested, this is null safe

Reply:
Can you post the local build result?

Reply:
actual it failed the build because the code style check, re-formatted code and pass the build, thanks for tips

Reply:
> Can you post the local build result?

actual it failed the build because the code style check, re-formatted code and pass the build, thanks for tips